samples/highcharts/accessibility/accessible-switch-theme/demo.js

```diff
@@ -0,0 +1,357 @@
+        },
+        errorbar: {
+            color: 'white'
+        },
+        map: {
+            nullColor: '#353535'
+        }
+    },
+
+    legend: {
+        backgroundColor: 'transparent',
+        itemStyle: {
+            color: textBright
+        },
+        itemHoverStyle: {
+            color: '#FFF'
+        },
+        itemHiddenStyle: {
+            color: '#606063'
+        },
+        title: {
+            style: {
+                color: '#D0D0D0'
+            }
+        }
+    },
+
+    credits: {
+        style: {
+            color: textBright
+        }
+    },
+
+    labels: {
+        style: {
+            color: '#707073'
+        }
+    },
+
+    drilldown: {
+        activeAxisLabelStyle: {
+            color: textBright
+        },
+        activeDataLabelStyle: {
+            color: textBright
+        }
+    },
+
+    navigation: {
+        buttonOptions: {
+            symbolStroke: '#DDDDDD',
+            theme: {
+                fill: '#505053'
+            }
+        }
+    },
+
+    rangeSelector: {
+        buttonTheme: {
+            fill: '#505053',
+            stroke: '#000000',
+            style: {
+                color: '#eee'
+            },
+            states: {
+                hover: {
+                    fill: '#707073',
+                    stroke: '#000000',
+                    style: {
+                        color: textBright
+                    }
+                },
+                select: {
+                    fill: '#303030',
+                    stroke: '#101010',
+                    style: {
+                        color: textBright
+                    }
+                }
+            }
+        },
+        inputBoxBorderColor: '#505053',
+        inputStyle: {
+            backgroundColor: '#333',
+            color: textBright
+        },
+        labelStyle: {
+            color: textBright
+        }
+    },
+    navigator: {
+        handles: {
+            backgroundColor: '#666',
+            borderColor: '#AAA'
+        },
+        outlineColor: '#CCC',
+        maskFill: 'rgba(180,180,255,0.2)',
+        series: {
+            color: '#7798BF',
+            lineColor: '#A6C7ED'
+        },
+        xAxis: {
+            gridLineColor: '#505053'
+        }
+    },
+
+    scrollbar: {
+        barBackgroundColor: '#808083',
+        barBorderColor: '#808083',
+        buttonArrowColor: '#CCC',
+        buttonBackgroundColor: '#606063',
+        buttonBorderColor: '#606063',
+        rifleColor: '#FFF',
+        trackBackgroundColor: '#404043',
+        trackBorderColor: '#404043'
+    }
+};
+
+function makeArray(length, populate) {
+    return [...new Array(length)].map(populate);
+}
+
+function makeChart(constructor, type, series) {
+    const capitalizeString = s => s.charAt(0).toUpperCase() + s.slice(1);
+    return Highcharts[constructor](`container-${type}`, Highcharts.merge(({
+        chart: {
+            type: type
+        },
+        title: {
```

Comment:
Would add exporting.enabled = false to the options here - no need for export menu on these